dist/origin-web-common.js

```diff
@@ -896,44 +896,6 @@ angular.module('openshiftCommonUI')
       }
     };
   }]);
```

Comment:
Pulling this out into the `test/fixtures` dir.... was it supposed to end up in the dist?